modules/crud/crud.go

```diff
@@ -57,7 +57,8 @@ func initBlock(dbType utils.DBType, enabled bool, connection string) (Crud, erro
 
 	case utils.MySQL, utils.Postgres:
 		return sql.Init(dbType, enabled, connection)
-
```

Comment:
no need to create another case
just add utils.MsSQL in the above case after utils.Postgres